sympy/series/formal.py

```diff
@@ -972,8 +980,19 @@ def infinite(self):
 
         return self.ind + inf_sum
 
+    @property
+    def fps_sym(self):
+        """Returns whether the function contains symbolic terms"""
+        ind = self.ind
+        s = list(ind.free_symbols.difference({self.x}))
```

Comment:
Can the conversion to `list` be avoided here as `set` also supports `len`?

Reply:
True.